frontend/src/scenes/early-access-features/earlyAccessFeatureLogic.ts

```diff
@@ -38,19 +38,25 @@ export const earlyAccessFeatureLogic = kea<earlyAccessFeatureLogicType>([
         actions: [earlyAccessFeaturesLogic, ['loadEarlyAccessFeatures', 'loadEarlyAccessFeaturesSuccess']],
     })),
     actions({
+        setEarlyAccessFeatureMissing: true,
         toggleImplementOptInInstructionsModal: true,
         cancel: true,
         editFeature: (editing: boolean) => ({ editing }),
         updateStage: (stage: EarlyAccessFeatureStage) => ({ stage }),
         deleteEarlyAccessFeature: (earlyAccessFeatureId: EarlyAccessFeatureType['id']) => ({ earlyAccessFeatureId }),
         setActiveTab: (activeTab: EarlyAccessFeatureTabs) => ({ activeTab }),
     }),
-    loaders(({ props }) => ({
+    loaders(({ props, actions }) => ({
         earlyAccessFeature: {
             loadEarlyAccessFeature: async () => {
                 if (props.id && props.id !== 'new') {
-                    const response = await api.earlyAccessFeatures.get(props.id)
-                    return response
+                    try {
+                        const response = await api.earlyAccessFeatures.get(props.id)
+                        return response
+                    } catch (error: any) {
+                        actions.setEarlyAccessFeatureMissing()
+                        throw error
+                    }
```

Comment:
This solution is totally fine for now. Only thing is, I think the better long term solution is to have the state object start in a `null` state. I think this is hard here as you use the same state for the kea form which would get in the way as it starts with the "new" state. Just an FYI for the future.